vhdl_lang/src/ast/visitor.rs

```diff
@@ -1645,26 +1610,6 @@ impl ASTNode for WaitStatement {
     }
 }
 
-impl ASTNode for FunctionSpecification {
```

Comment:
See my other blocking comment.

Reply:
For `FunctionSpecification ` I also added the designator as a child, since it is mentioned in `ProcedureSpecification` as well 🤷